qemu/tests/cfg/virtio_fsd_check_info.cfg

```diff
@@ -1,5 +1,6 @@
 - virtio_fsd_check_info:
     no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8.u0 Host_RHEL.m8.u1
```

Comment:
Still remove `Host_RHEL.m6 Host_RHEL.m7` here, right?

Reply:
sorry for the issue, updated again